### DISCUSSION TITLE
audio setting for focusride scarlett solo

### DISCUSSION THREAD

**Discussion by ale211734** [jamulussoftware/jamulus#3458] [Technical Support & Help]
Hello, when I go to the audio settings of my sound card, a Focusrite Scarlett Solo, it doesn’t give me the option to change the input channel. I have 2 input channels, but the program doesn’t allow me to select them. I’m attaching the screenshot.
![focusRide](https://github.com/user-attachments/assets/f2ac6f33-5be5-4392-9882-7e4e5fdc53d9)

**Comment by ann0see** [comment]
If it's just two channels they should be selected automatically.
One left one right. 
The client can mix one channel up to two by enabling mono in/stereo out. 

Since you are in mono mode, I think both should be mixed by default into one channel. And you'll also only get one channel back from the server which gets distributed to your output device.

**Reply by dcorson-ticino-com** [reply]
Note that Jamulus treats 2 channels as a stereo pair.
It is not possible to use them as 2 mono channels.
